DuckDuckGo/SiteRatingView.swift

```diff
@@ -75,14 +75,18 @@ public class SiteRatingView: UIView {
             return
         }
         
-        let grades = siteRating.scores
-        let grade: Grade.Score
+        guard !AppUrls().isDuckDuckGoSearch(url: siteRating.url) else {
```

Comment:
I believe this will need to check for the `.removeSERPHeader` variant, I currently see the Dax icon on DuckDuckGo searches without the variant active.

Reply:
@samsymons thanks, this should be fixed now!